src/style/components/_text.styl

```diff
@@ -0,0 +1,3 @@
+.c-text {
+    white-space: pre;    
```

Comment:
Maybe use pre-wrap here? (Text will wrap when necessary, and on line breaks). So it won't overflow when placed inside container.